src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -114,4 +118,53 @@ module sonic-types {
             pattern "percentage|used|free|PERCENTAGE|USED|FREE";
         }
     }
+
+    typedef oper-status {
+        type enumeration {
+            enum unknown;
+            enum up;
```

Comment:
oper-status field is for operational state of any interface or port. So it will not be used for CONFIG_DB. It will be used for non CONFIG_DB fields. Currently following tables modelled in sonic yang are using this field MCLAG_TABLE, MCLAG_REMOTE_INTF_TABLE, PORT_TABLE, LAG_TABLE, VLAN_TABLE. If any table has to use in UPPER case, it can be added.